config/formats.ts

```diff
@@ -1049,6 +1049,7 @@ export const Formats: (FormatsData | {section: string, column?: number})[] = [
 				if (validSources.includes(evoFamily)) continue;
 
 				set.species = donorSpecies.name;
+				set.name = donorSpecies.baseSpecies;
```

Comment:
```suggestion
				if (!set.name) set.name = donorSpecies.baseSpecies;
```

This seems safer to me?